src/cpp/fastdds/subscriber/SubscriberImpl.hpp

```diff
@@ -23,16 +23,16 @@
 
 #include <fastrtps/attributes/SubscriberAttributes.h>
 
+#include <fastdds/dds/core/status/StatusMask.hpp>
 #include <fastdds/dds/subscriber/DataReaderListener.hpp>
 #include <fastdds/dds/subscriber/qos/DataReaderQos.hpp>
 #include <fastdds/dds/subscriber/qos/SubscriberQos.hpp>
-#include <fastdds/dds/core/status/StatusMask.hpp>
 #include <fastrtps/types/TypesBase.h>
 
 #include <statistics/rtps/monitor-service/interfaces/IStatusQueryable.hpp>
 
-#include <mutex>
 #include <map>
+#include <mutex>
```

Comment:
The header order is not only alphabetical. First the STL headers, then external library headers and finally header of Fast DDS separated between public and private headers.